splitgraph/core/fragment_manager.py

```diff
@@ -517,6 +501,53 @@ def _get_patch_fragment_hashes_stats(
         object_id = "o" + sha256((content_hash + schema_hash).encode("ascii")).hexdigest()[:-2]
         return deletion_hash, insertion_hash, object_id, rows_inserted, rows_deleted
 
+    def _store_object_from_temp_table(
+        self,
+        object_id: str,
+        tmp_table_id: str,
+        table: "Table",
+        in_fragment_order: Optional[List[str]] = None,
+        overwrite: bool = False,
+    ):
```

Comment:
(IIRC mypy might not actually descend into this method to typecheck it otherwise)

```suggestion
    ) -> None
```